crates/full-node/full-node-configs/src/sequencer.rs

```diff
@@ -267,15 +274,31 @@ pub struct TimingOracleConfig {
     pub private_key_hex: Option<String>,
 }
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, Eq, PartialEq, JsonSchema)]
+pub struct SovRateLimiterConfig<Address: Copy> {
+    pub default_limits: Limits,
+    pub address_custom_limits: Vec<(Address, Limits)>,
```

Comment:
We will set the custom limits in the next PR. This PR adds only structural changes. The rate limiter is still diabled in prod.